packages/flame_forge2d/lib/forge2d_camera.dart

```diff
@@ -1,16 +1,36 @@
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
 import 'package:flame/game.dart';
 
+import 'flame_forge2d.dart';
+
 class Forge2DCamera extends Camera {
+  Body? _targetBody;
+
+  /// Used to minimize the amount of [Vector2] objects created
+  Vector2? _unprojectVector;
+  Vector2? _projectVector;
```

Comment:
Is it meaningful to allow these to be null?

Reply:
If they aren't used they won't have to be initialized, but since it is only two I think you're probably right, we should just initialize them and simplify the logic slightly where they are used.